test/projectLinks.test.js

```diff
@@ -36,4 +37,36 @@ describe('Project Link', () => {
         .where({ project_id: 'projectLink-Int' })
         .del());
   });
+
+  it('@integration: returns an array of project links', async () => {
+    const driver = new MySQLDriver();
+    const context = { project: new Project('CKQ225WSV') };
+    const projectLinkInstances = await ProjectLink.findByProject(context.project, driver);
+    expect(projectLinkInstances).to.be.an('array');
+  });
+
+  it('@integration: findByProject retrieves project links from db', async () => {
+    const driver = new MySQLDriver();
+    const context = { project: new Project('CKQ225WSV') };
+    const projectLinkInstances = await ProjectLink.findByProject(context.project, driver);
+
+    const expectedResult = [
+      {
```

Comment:
This looks like you are depending on data added to your own database. I do not have these so the tests are failing.

I would recommend you test to the edge of the system which is `MySQLDriver` in this case, asserting that `getAllLinksByProject` is called correctly.

Reply:
got it!